docs/how-to/installation/docker.txt

```diff
@@ -34,16 +33,57 @@ Preliminary Setup
       docker pull determinedai/determined-master:VERSION
       docker pull determinedai/determined-agent:VERSION
 
+#. Assuming you are running multiple containerized services on the Determined
+   master, optionally create the ``determined`` network that we will use in the
```

Comment:
blocking: I think it would be helpful to distinguish "master" (the machine) from "master" (the program), probably by saying "master node/machine/instance" (pick one) or something when you mean the former.

Reply:
Done; I added a bunch of distinction between master / agent "instance" and "container" throughout